recipes/numpy_indexed/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: numpy-indexed
```

Comment:
Another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).